packages/slice-machine/src/models/slice/context.tsx

```diff
@@ -69,15 +69,19 @@ const SliceProvider: React.FunctionComponent<SliceProviderProps> = ({
   );
 };
 
-export const SliceHandler = ({ children }: { children: any }) => {
+export const SliceHandler = ({
```

Comment:
type the const with React.FunctionComponent, with this you can remove the children type :)